RELEASE_NOTES.md

```diff
@@ -3,7 +3,8 @@
 ## in develop
 
 * Command file is now echoed to stderr rather than stdout
-
+* Updates build environment to SBT 1.6.1
```

Comment:
I don't think we generally put changes that only affect the dev environment into the changelog. @YuxinShi0423 @jdidion

Reply:
Also, whitespace between end of list & next heading

Reply:
Removed. Thanks!